guacamole/src/main/webapp/app/login/loginModule.js

```diff
@@ -23,4 +23,4 @@
 /**
  * The module for the login functionality.
  */
-angular.module('login', ['element']);
+angular.module('login', ['element', 'navigation']);
```

Comment:
I thought we wanted the "home" button to be smart and redirect to the right place, otherwise we have to redirect to /, and then the homeController pulls down all the permissions and groups to determine if it's the correct place, then possibly redirecting. That seems a bit inefficient.